modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-butane_{context}"]
+= Configuring machine configs with Butane
+
+Ignition config files, which control the RHCOS configuration, are used to deploy control plane and worker machines. A machine config uses an Ignition config to tell machines to create users and filesystems, set up the network, install systemd units, and more.
+
+Ignition configs are machine-readable JSON files that can be difficult to modify. You can use human-readable Butane configs to create Ignition configs, making the machine configuration process much easier.
+
+== About Butane
+
+Butane is a human-readable YAML utility that is used by {op-system} to automatically write many of the labor-intensive parts of Ignition for you. The Butane tool, based on the `butane` specification, consumes a Butane config and produces an Ignition config that can be given to a {op-system} machine when it first boots.
+
+[id="rhcos-installing-butane_{context}"]
+== Installing Butane
+
+You can install the Butane tool (`butane`) to create machine configs with {op-system} from a command-line interface. You can install `butane` on Linux, Windows, or macOS by downloading the related binary.
+
+[TIP]
+====
+Butane releases are backwards-compatible with older releases and with the Fedora CoreOS Config Transpiler (FCCT), unless otherwise noted.
+====
+
+.Procedure
+
+. Navigate to the Butane image download page at https://mirror.openshift.com/pub/openshift-v4/clients/butane/.
+. Get the `butane` binary:
+.. For the newest version of Butane, get the latest `butane` image:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane
+----
++
+.. Optional: For a specific type of architecture you are installing on, such as `butane-aarch64` or `butane-ppc641e`, indicate the appropriate URL. For example:
++
+[source,terminal]
+----
+$ curl https://mirror.openshift.com/pub/openshift-v4/clients/butane/latest/butane-aarch64
+----
++
+. Unpack and unzip the archive.
```

Comment:
It's not clear to me what to do from this point. I'm using steps from [Installing the OpenShift CLI](https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html) for a workflow because I couldn't find an example for using the mirror site. Not sure what format the butane file is in. I got it downloaded using podman pull but my understanding is that we aren't using that in 4.8. 

Need some clarification of this process, please.